app/views/editor/posts/_form.html.haml

```diff
@@ -22,15 +22,17 @@
 
 :javascript
   $(function() {
-    function applySelect2($element) {
+    function applySelect2($element, allowClear) {
       $element.select2({
-        placeholder: '選択してください'
+        placeholder: '選択してください',
+        allowClear: allowClear
       });
     }
 
-    applySelect2($('select'));
+    applySelect2($('select#post_category_id'), false);
```

Comment:
Please use select2's options instead of primitive value.

----

パッと見て、`true` ? `false` ? これどうなるの？というのが実装読むまでわかりませんでした。
単純に select2 に渡すオプションと API を合わせてあげるほうが読みやすくそしてメンテナブルだと感じました。

``` js
applySelect2($('select#post_category_id'), {allowClear : false});
```